pkg/helpers/crd.go

```diff
@@ -23,9 +23,12 @@ const (
 	protectionSettingEnvVar = "LIEUTENANT_DELETE_PROTECTION"
 )
 
+// DeletionState of an object
```

Comment:
While I appreciate the effort, adding this documentation brakes the keep PR small principle. I would have made PR of its own for them.